module/Company/src/Company/Model/CompanyJobPackage.php

```diff
@@ -90,4 +110,24 @@ public function removeJob(Job $job)
     {
         $this->jobs->removeElement($job);
     }
+
+    /**
+     * Adds a approvalVacancy to the package.
+     *
+     * @param ApprovalVacancy $approvalVacancy approvalVacancy to be added
+     */
+    public function addApprovalVacancy(ApprovalVacancy $approvalVacancy)
+    {
+        $this->approvalVacancies->add($approvalVacancy);
```

Comment:
Probably advisable to put a check on this (i.e. check if the `$approvalVacancy` already in the `ArrayCollection`).